src/frontend/src/flows/register/passkey.ts

```diff
@@ -17,14 +17,17 @@ const savePasskeyTemplate = ({
   construct,
   i18n,
   cancel,
+  scrollTo = false,
 }: {
   construct: () => void;
   i18n: I18n;
   cancel: () => void;
+  /* put the CTA into view */
```

Comment:
What is "CTA"?

Reply:
[Call-to-Action](https://en.wikipedia.org/wiki/Call_to_action_(marketing))